### ISSUE TITLE
ImportError: cannot import name 'dense_features'

### ISSUE THREAD

**Issue by shari-amalgamrx** [re-search/DocProduct#29]
Unable to import the package. The solution seems to be to update the tf but then its throwing an error that the DocProduct doesn't support tf==2.1

Failed to load GPU Faiss: No module named 'faiss.swigfaiss_gpu'
Faiss falling back to CPU-only.
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-5-4d43851f3eef> in <module>()
      1 #@title Import and initalize the fine-tuned GPT2 generator. Double-click to see code.
      2 
----> 3 from docproduct.predictor import GenerateQADoc
      4 
      5 pretrained_path = 'BioBertFolder/biobert_v1.0_pubmed_pmc/'

8 frames
/usr/local/lib/python3.6/dist-packages/tensorflow_estimator/python/estimator/canned/dnn.py in <module>()
     21 import six
     22 
---> 23 from tensorflow.python.feature_column import dense_features
     24 from tensorflow.python.feature_column import dense_features_v2
     25 from tensorflow.python.feature_column import feature_column

ImportError: cannot import name 'dense_features'

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------

**Comment by Santosh-Gupta** [comment]
faiss is seperate from Tensorflow, did you double check to make sure it's being installed ok?

**Comment by shari-amalgamrx** [comment]
@Santosh-Gupta Yes I did install the updated version. Still unable to run the colab? Do you by any chance happen to have a step by step instruction to run the notebook? That would be really helpful. 
Thank you

**Comment by Santosh-Gupta** [comment]
It should run by executing the cells in order. Which notebook are you trying, there were a couple in the readme.

**Comment by shari-amalgamrx** [comment]
I was able to run the model that just retrieves a sentence. Even that had some dependency issues. But I am unable to run the notebook that uses gpt2

**Comment by Santosh-Gupta** [comment]
ahh, that's @ash3n 's , maybe he can give some insight.
